packages/fxa-auth-server/lib/payments/paypal-client.ts

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import querystring from 'querystring';
+import superagent from 'superagent';
```

Comment:
This jumped out at me on first glance because I know that auth-server already talk to other services.  But looks like those all buried behind `createBackendServiceAPI` (which in turn uses poolee), so this addition is 👍 with me.

(But if `createBackendServiceAPI` also works for this, then I'd prefer the consistency.)

Reply:
Why would we not want to use `createBackendServiceAPI` here? Is it because this API is for PayPal servers rather than FxA servers (i.e. we don't need to declare routes here, but rather in the higher level PayPal Helper object in https://jira.mozilla.com/browse/FXA-2878)?